src/common/Utilities.py

```diff
@@ -55,9 +55,41 @@ def examples_data_path(data_type):
         return os.path.join(SIRF_INSTALL_PATH , data_path)
     elif SIRF_PATH is not None:
         return os.path.join(SIRF_PATH, 'data', 'examples', data_type)
-    else:
-        errorMsg = 'You need to set the SIRF_DATA_PATH or SIRF_INSTALL_PATH environment variable to allow finding the raw data.'
-        raise ValueError(errorMsg)
+
+    # one final effort
+    try:
+        # Try find <SIRF_INSTALL_PATH>/share/data/examples
+        py_root = os.path.abspath(sirf.__file__)
```

Comment:
if I am not mistaken, you never use `py_root` except in `py_root.split(os.sep)` - I suggest replacing this with
```
py_root_split = os.path.abspath(sirf.__file__).split(os.sep)
```